src/core/EventDispatcher.js

```diff
@@ -93,6 +93,21 @@ class EventDispatcher {
 
 	}
 
+	/**
+	 * Removes all event listeners for the given event type.
+	 *
+	 * @param {string} type - The type of event.
+	 */
+	removeEventListeners( type ) {
+
+		const listeners = this._listeners;
+
+		if ( listeners === undefined ) return;
+
+		delete this._listeners[ type ];
+
+	}
+
```

Comment:
@linbingquan Can we elaborate on some practical use cases for this function? It seems more like a workaround for applications not managing their own event registration well.

It should be noted that adding this function means that EventDispatcher is becoming out of sync with the API of the browsers event system, which I believe was the original intent of the class. This might be okay but perhaps deserves some consideration.